apps/dapp/src/components/Metrics/Metrics.tsx

```diff
@@ -41,7 +41,7 @@ const Metrics = ({ treasuryMetrics, isHome, alignCenter }: MetricsProps) => {
           cryptoName={'vAPY'}
           value={`${formatNumber(dynamicApy)}%`}
           imageData={{
```

Comment:
See other comment about making this an optional prop

Reply:
https://github.com/TempleDAO/temple/pull/576/commits/6af1e9ea10513b33c23dfc47285cb125663b5969